src/neo-asm-cli/CommandLineOption.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace neo_asm_cli
+{
+    class CommandLineOption
+    {
+        public List<string> inputfiles;
+        public string outputfile;
+
+        public static CommandLineOption Parse(string[] args)
+        {
+            if (args == null || args.Length == 0)
+                throw new Exception("error params");
+            CommandLineOption option = new CommandLineOption();
+            option.inputfiles = new List<string>();
+            option.outputfile = null;
+            for (var i = 0; i < args.Length; i++)
+            {
+                if (args[i][0] != '-') continue;
+
+                if (args[i] == "-i")
+                {
+                    option.inputfiles.Add(args[i + 1]);
```

Comment:
If is the last argument, it could fail

I recommend to use https://github.com/commandlineparser/commandline for parse arguments.

Reply:
Woops, posted my comment before noticing this one. DOn't really care which library we use, as long as we don't roll our own